app/views/layouts/base.html.erb

```diff
@@ -1,6 +1,7 @@
 <% mascot = current_user&.mascot || 'ki' %>
+<% pervert = current_user&.pervert || false %>
 <!DOCTYPE html>
-<html class="<%= current_user ? "force-#{current_user.colour_preference}" : "" %>">
+<html class="<%= current_user ? "force-#{current_user.colour_preference}" : ""%> <%= "current-mascot-"+mascot %> <%=pervert ? "currently-perverted" : ""%> ">
```

Comment:
See the comment in _details.html.erb.

This is a workaround for not having access to current_user when rendering partials via Turbo, for cases where our only need is to style the document based on the current mascot.

Reply:
Bonus: Custom user CSS can be aware of the current lizard/perv combo of the user by leveraging this